.circleci/config.yml

```
@@ -303,7 +303,7 @@ commonYAMLStructure:
       name: deploy to test pypi
       command: |
         . venv/bin/activate
-        ./scripts/deploy test.pypi --no-publish
+        ./scripts/deploy test.pypi --no-publish --bump-version dev
```

```suggestion
        ./scripts/deploy test.pypi --no-publish
```
This does not need to be changed. We just build package locally

---

The script has the required for either version or bump-version, so it has to be there or the script won't run.